packs/ezoic.js

```diff
@@ -0,0 +1,59 @@
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits.*/
+  'total-byte-weight': '',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits. Ezoic Leap is Ezoic's site speed improvement toolset. Critical CSS is a setting name.*/
+  'render-blocking-resources': 'Using [Ezoic Leap](https://pubdash.ezoic.com/speed) and enabling `Critical CSS` and `Script Delay` will defer non-critical JS/CSS.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits. Ezoic Leap is Ezoic's site speed improvement toolset. Minify CSS is a setting name.*/
+  'unminified-css': 'Using [Ezoic Leap](https://pubdash.ezoic.com/speed) and enabling `Minify CSS` will automatically minify your CSS to reduce network payload sizes.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits. Ezoic Leap is Ezoic's site speed improvement toolset. Minify Javascript is a setting name.*/
+  'unminified-javascript': 'Using [Ezoic Leap](https://pubdash.ezoic.com/speed) and enabling `Minify Javascript` will automatically minify your JS to reduce network payload sizes.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits.*/
+  'efficient-animated-content': '',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits.*/
+  'unused-javascript': '',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits. Ezoic Leap is Ezoic's site speed improvement toolset. Efficient Static Cache Policy is a setting name.*/
+  'uses-long-cache-ttl': 'Using [Ezoic Leap](https://pubdash.ezoic.com/speed) and enabling `Efficient Static Cache Policy` will set recommended values in the caching header for static assests.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `Ezoic`. This is displayed after a user expands the section to see more. No character length limits. Ezoic Leap is Ezoic's site speed improvement toolset. Next-Gen Formats is a setting name.*/
+  'uses-optimized-images': 'Using [Ezoic Leap](https://pubdash.ezoic.com/speed) and enabling `Next-Gen Formats` will convert images to WebP.',
```

Comment:
for this string and all others, change up the grammar to be more like:

`Use Ezoic Leap and enable `Some-Feature` to convert images to WebP`

basically, avoid present participles (adding -ing to verbs) and prefer the normal form of verb, as it tends to simplify the language